__tests__/app.test.js

```diff
@@ -35,13 +35,57 @@ describe("/api/topics", () => {
         .expect(200)
         .then((response) => {
           const topics = response.body.topics;
-            expect(topics.length).toBe(3)
-            topics.forEach((topic) => {
-              expect(topic).toHaveProperty("description");
-              expect(topic).toHaveProperty("slug");
-            });
-          
+          expect(topics.length).toBe(3);
+          topics.forEach((topic) => {
+            expect(topic).toHaveProperty("description");
+            expect(topic).toHaveProperty("slug");
+          });
         });
     });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  describe("GET:", () => {
+    describe("status: 200", () => {
+      test("responds with article object", () => {
+        return request(app)
+          .get("/api/articles/1")
+          .expect(200)
+          .then((response) => {
+            const article = response.body.article;
+            const expected = {
+              article_id: 1,
+              article_img_url:
```

Comment:
can we use a more flexible matcher here? like expect.objectContaining or .toMatchObject